src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -6,6 +6,8 @@
 import htsjdk.variant.vcf.VCFHeader;
 import org.apache.avro.generic.GenericRecord;
 import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang.math.LongRange;
+import org.apache.hadoop.yarn.webapp.hamlet2.Hamlet;
```

Comment:
To be or not to be? Here, this should not be.

Reply:
🗣️💀